src/app/Annotation.tsx

```diff
@@ -151,6 +154,27 @@ const Annotation: React.FC<AnnotationProps> = (props) => {
         <TranslucentBox nodeRef={ref} css={{ cursor: "inherit", ...css }} />
       );
     }
+    case "SELECT": {
+      const isSelected = Boolean(selectedAnnotations[props.id]);
+      return (
+        <TranslucentBox
+          nodeRef={ref}
+          onClick={(e) => {
```

Comment:
If `isSelected` is true, I don't think `e.stopPropagation()` will ever get called. Was this intentional?